src/less/components/form.less

```diff
@@ -187,18 +173,6 @@ legend {
     padding: @input-padding-y 0;
 }
 
-.form-control-focus(@color: @input-border-focus) {
-    @color-rgba: rgba(red(@color), green(@color), blue(@color), 1);
-
-    &:focus {
-        // outline: 0;
-        color: @text-color;
-        outline-color: @color-rgba;
-        box-shadow: none;
-        background-color: @white;
-    }
-}
-
 .placeholder(@color) {
```

Comment:
out of the scope of this ticket, but:
delete those placeholder browser specific rules and use the standard `::placeholder`, it seems way supported enough: https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder

Reply:
Nice catch 🔥